modules/main-rules/pages/roll-dice-and-add-firefight-bonuses.adoc

```diff
@@ -1,6 +1,9 @@
-= FF3. Roll dice and add firefight bonuses
+= FF3. Make the combat resolution roll
 
+.Combat resolution roll
 Both players: Roll a D6 and add your firefight bonuses to the result.
 
 * The player with the highest total score wins the firefight.
-* If the total scores are the same then the outcome hangs in the balance! Place one Blast marker on every detachment in the firefight and then *roll again*.
\ No newline at end of file
+* If the total scores are the same then the outcome hangs in the balance! Place one Blast marker on every detachment in the firefight and then *roll again*.
```

Comment:
To mirror my edit for the equivalent close combat topic, `roll-dice-and-add-combat-bonuses.adoc`:
```suggestion
* If the total scores are the same then the outcome hangs in the balance!
Place one Blast marker on every detachment in the combat, and then proceed again from xref:work-out-firefight-bonuses.adoc[].
```